geoviews/tests/plotting/mpl/test_chart.py

```diff
@@ -0,0 +1,197 @@
+import numpy as np
+import xarray as xr
+import geoviews as gv
+
+from geoviews.element import WindBarbs
+
+from holoviews.tests.plotting.matplotlib.test_plot import TestMPLPlot
+from holoviews.tests.plotting.utils import ParamLogStream
+
+from geoviews import Store
+
+gv.extension("matplotlib")
```

Comment:
Thanks for catching that; I simply copied over from my notebook.

Reply:
Okay, I don't think TestMPLPlot works without gv.extension because it's calling the HoloViews store instead of the GeoViews store.
```

        try:
            plotclass = Store.registry[cls.backend][element_type]
        except KeyError:
>           raise SkipRendering(f"No plotting class for {element_type.__name__} found.")
E           holoviews.core.options.SkipRendering: No plotting class for WindBarbs found.
```